Library/ViewModels/AddNewCardViewModelTests.swift

```diff
@@ -12,12 +12,13 @@ internal final class AddNewCardViewModelTests: TestCase {
   private let activityIndicatorShouldShow = TestObserver<Bool, NoError>()
   private let addNewCardFailure = TestObserver<String, NoError>()
   private let addNewCardSuccess = TestObserver<String, NoError>()
+  private let cardBrandUnsupportedErrorMessageHiddenObserver = TestObserver<Bool, NoError>()
   private let cardholderNameBecomeFirstResponder = TestObserver<Void, NoError>()
   private let dismissKeyboard = TestObserver<Void, NoError>()
   private let cardholderName = TestObserver<String, NoError>()
   private let cardNumber = TestObserver<String, NoError>()
-  private let cardExpMonth = TestObserver<Int, NoError>()
-  private let cardExpYear = TestObserver<Int, NoError>()
+  private let cardExpMonth = TestObserver<UInt, NoError>()
```

Comment:
Also, just realizing now, this could be a great opportunity to introduce `Month` `typealias` or `Year` `typealias` that would make it more readable.